repository/r.json

```diff
@@ -882,6 +882,17 @@
 			]
 		},
 		{
+			"name": "ReqsLang",
+			"details": "https://github.com/petarslovic/sublime-reqs-language",
+			"labels": ["requirements", "todo"],
```

Comment:
Sorry, I meant literally the label `language syntax`: https://packagecontrol.io/browse/labels/language%20syntax